Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM torosent/qpid
+
+RUN git clone -b EventProcessingHostArchitecture --single-branch https://github.com/CatalystCode/azure-event-hubs-python.git
```

Comment:
Change this line to clone the  original repo

Reply:
Done